zoo-css-challenge/index.html

```diff
@@ -100,7 +104,7 @@ <h3>Learning resources for schools</h3>
 			</div>
 		</section>
 
-		<section class="membership">
+		<section class="membership" id="member-container">
```

Comment:
```suggestion
		<section class="membership">
```
As there is already a `.membership` class here, we can use that class for styling so we don't need to add `id="member-container"`